simplecpp.cpp

```diff
@@ -377,6 +377,42 @@ class StdIStream : public simplecpp::TokenList::Stream {
     std::istream &istr;
 };
 
+class StdCharBufStream : public simplecpp::TokenList::Stream {
```

Comment:
Is this declared in a unnamed namespace?

Reply:
ok maybe we should start allowing c++11 sometime soonish then.

if we want to keep it c++03 compatible we could hide the `namespace {` in a ifdef but I don't know if that is necessary.

But let's look at c++11 in another PR.